src/config.rs

```diff
@@ -53,6 +53,7 @@ const LOCKCF_MAX_MEM: usize = GB as usize;
 const RAFT_MIN_MEM: usize = 256 * MB as usize;
 const RAFT_MAX_MEM: usize = 2 * GB as usize;
 const LAST_CONFIG_FILE: &str = "last_tikv.toml";
+const MAX_BLOCK_SIZE: usize = 32 * MB as usize;
```

Comment:
if we want to enlarge `block_size` for some purpose, is this threshold too small?

Reply:
IMO, I guess 32MB is too huge.